### PR TITLE
feature/SKARE-39-identify-two-pair

### PR DESCRIPTION
**Pull request by erinyooey**

### CODE REVIEW

**Review by robcas12 — APPROVED** [review]
Requirement Satisfaction: The identification class and its two-pair function meet the requirements outlined in the Trello board. Great Job.

Logic Errors: No significant logic errors within the two-pair function.

Test Coverage: This could be expanded to test for edge cases like full houses or four of a kind, which may include pairs in their structure. This can be a task added for a future sprint.

Style Guides: No major style guide issues.

**Comment by erinyooey** [comment]
Thanks for the detailed review!

**Review by Agwebberley — APPROVED** [review]

File: source/CardGames.py [hunk 1]
```diff
@@ -343,4 +346,32 @@ def checkBets(self):
...
+    return max(self.bets) - self.bets[idx]
+  
+class Identification:
+  def __init__(self):
+    self.hand = [] # Hold values of cards in hand
+    self.name = "" # Name defined by the rank definition functions
+
+  # Include functions that return what defines the various poker hand ranks.
+  
+  def twoPair(self):
```
[the remark this answers is not in the document]
Comment (erinyooey) [reply source/CardGames.py:358]: That's a good idea! Done!
